src/cliEntry.js

```diff
@@ -160,11 +159,9 @@ async function run(args: Array<string>) {
 
   const opts = command.options || [];
 
-  const { _, ...flags } = camelcaseKeys(
```

Comment:
Why removing this?

Reply:
`camelcaseKeys` renamed `hot-reloading` to `hotReloading`. This later fails in `validateOptions()` because we're trying to parse a flag that doesn't exist ([see line 97](https://github.com/callstack/haul/blob/ee773d160861d2831bf28512322a680fa970668a/src/cliEntry.js#L97)).

Reply:
Aha, I see where this is used now… Let me try and fix `validateOptions()` instead.

Reply:
Okay, I think I just misunderstood how to name the options. It's fixed now. Thanks for catching this :)